doc/nrf/libraries/networking/downloader.rst

```diff
@@ -187,12 +187,27 @@ The following snippet shows how to download a file using HTTPS:
 Limitations
 ***********
 
-The library requires the host server to provide a Content-Range field in the HTTP GET response header when using HTTPS with the nRF91 Series devices.
```

Comment:
An intro sentence like below would be good.

_The following limitations apply to this library:_